HDFS is unsupported (#64126)
ClickHouse/ClickHouse: docs/en/operations/storing-data.md

Zero-copy replication is disabled by default in ClickHouse version 22.8 and higher.  This feature is not recommended for production use.
:::

## Using HDFS storage (Unsupported)

In this sample configuration:
- the disk is of type `hdfs` (unsupported)
- the data is hosted at `hdfs://hdfs1:9000/clickhouse/`

> By the way, HDFS is unsupported and won't work.

Reviewer: ```suggestion
By the way, HDFS is unsupported and therefore there might be issues when using it. Feel free to make a pull request with the fix if the issues arises.
```

</clickhouse>
```

> Keep in mind that HDFS does not work at all.

Reviewer: May be this message is too much? "At all" is not really true... If if did not work at all, there would have been no sense having hdfs in documentation at all.

Reviewer 2: @kssenii latest HDFS have a hard limit 1million files per folder. 
Clickhouse stores all objects in the the single HDFS folder (for HDFS as external storage, unlike S3).
The average table with 500 columns reaches this limit 1million objects easily, it's less 1000 active+inactive wide parts, after that a table is unable to insert or to merge.

https://github.com/ClickHouse/ClickHouse/issues/43112

> Keep in mind that HDFS does not work at all.

Author: ```suggestion
Keep in mind that HDFS may not work in corner cases.
```
